tutorials/ios-chatbot/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: How to build a conversational app using Cloud Machine Learning APIs (Part 1 of 3)
+description: How to build a conversational app using Cloud Machine Learning APIs (Part 1 of 3).
+author: PokerChang
+tags: Cloud Functions, Dialogflow, API.AI, Webhooks, Localization, Chatbot, Machine Learning API, Transation, Vision, Speech
+date_published: 2017-01-02
+---
+
+For consumers, conversational apps (such as chatbot) are among the most visible examples of machine learning in action. For developers, building a conversational app is instructive for understanding the value that machine-learning APIs bring to the process of creating completely new user experiences.
+
+In this three-part post, we will show you how to build an example "[tour guide](https://youtu.be/qDAP3ZFjO48)" app for Apple iOS that can see, listen, talk and translate via [API.AI](https://api.ai/) (a developer platform for creating conversational experiences) and [Google Cloud ML APIs for Speech, Vision and Translate](https://cloud.google.com/products/machine-learning/). You will also see how easy it is to support multiple languages on these platforms. 
+
+[![English Demo](http://img.youtube.com/vi/qDAP3ZFjO48/0.jpg)](https://youtu.be/qDAP3ZFjO48)
+
+The three parts will focus on the following topics:
+
+Part 1
+
+
+
+*   Overview
+*   Architecture
+*   API.AI intents
+*   API.AI contexts
+
+[Part 2]
+
+*   API.AI webhook with [Cloud Functions](https://cloud.google.com/functions/)
+*   [Cloud Vision API](https://cloud.google.com/vision/)
+*   [Cloud Speech API](https://cloud.google.com/speech/)
+*   [Cloud Translation API](https://cloud.google.com/translate/)
+*   Support multiple languages
+
+[Part 3]
+
+*   Support the Google Assistant via Actions on Google Integration
+
+This post is Part 1 and Part 2 and 3 will be published in the following weeks.
```

Comment:
Suggested change:
This post is Part 1, and Part 2 and Part 3 will be published in the following weeks.